src/main/java/seedu/address/logic/commands/InputParticipationCommand.java

```diff
@@ -21,6 +21,9 @@ public class InputParticipationCommand extends Command {
     public static final String SUCCESS_MSG = "Participation points input successfully!";
     public static final String ATTENDANCE_NOT_MARKED = "Before inputting participation points, "
             + "mark the attendance of the student first!";
+    public static final String ATTENDANCE_ABSENT = "The student was absent for this tutorial!";
+    public static final String PARTICIPATION_POINTS_OUT_OF_RANGE = "Participation points "
+            + "must be between 0 and 1000.";
```

Comment:
Do you think that, for code maintainability, we should define it this way instead?

```java
    public static final String PARTICIPATION_POINTS_OUT_OF_RANGE = "Participation points "
            + "must be between 0 and " + MAXIMUM_PARTICIPATION_POINTS + ".";
```